test/unit/test.spec.js

```diff
@@ -83,4 +84,40 @@ describe('Test', function() {
       expect(this._test.isPending(), 'to be', true);
     });
   });
+
+  describe('.markOnly()', function() {
+    var sandbox;
+
+    beforeEach(function() {
+      sandbox = sinon.createSandbox();
+    });
+
+    afterEach(function() {
+      sandbox.restore();
+    });
+
+    it('should call appendOnlyTest on parent', function() {
```

Comment:
you can combine this and the next into a single test via [unexpected-sinon](http://unexpected.js.org/unexpected-sinon/assertions/spy/to-have-a-call-satisfying/):

```js
it('should call appendOnlyTest on parent', function() {
  var test = new Test('foo');
  var spy = sandbox.spy();
  test.parent = {
    appendOnlyTest: spy
  };
  test.markOnly();
  // you might need to remove 'exhaustively' from the string
  expect(spy, 'to have a call exhaustively satisfying', [test]).and('was called once');
});

Reply:
Done!